doc/BASICS.md

```diff
@@ -0,0 +1,97 @@
+[Documentation index](../README.md#documentation)
+
+# Basic concepts
+
+gitbot runs a validation script, binary or command (-t or --test) against PRs of the GitHub repository you specify (-r or --repo) .
+
+There are two basic ways of using it:
+
+* It can run against the first untested PR or the first PR with a comment to force a test, or against the PR you specify.
+ 
+  In this case you will need to run gitbot as many times as opened PRs requiring tests.  
+  
+  It works this way so if you are using Jenkins pulling the repository, you can have one job build for each gitbot execution.
+
+* If you are using [webhooks](https://developer.github.com/webhooks/), then you just need to specify the ID of the Pull Requests that started the hook (--P or --PR)
+
+It is also posible to tell gitbot to ignore PRs unless specific files are changed (by path or by extension with -f or --file), and specify a URL to added to the Pull Requests with the link to the log with the test output, for example to a Jenkins log (-u o --url).
+
+# Current syntax:
+
+```
+Usage: gitbot.rb [options]
+
+Mandatory options:
+    -r, --repo 'REPO'                GitHub repository to look for PRs. For example: openSUSE/gitbot.
+    -c, --context 'CONTEXT'          Context to set on comment (test name). For example: python-test.
+    -d, --description 'DESCRIPTION'  Test decription
+    -t, --test 'TEST.SH'             Command, or full path to script/binary to be used to run the test.
+    -f, --file '.py'                 pr_file type to run the test against: .py, .rb
+    -g, --git_dir 'GIT_LOCAL_DIR'    Specify a location where gitbot will clone the GitHub project. If the dir does not exists, gitbot will create one. For example: /tmp/
+
+Optional options:
+    -C, --check                      Check if there is any PR requiring a test, but do not run it.
+        --changelogtest              Check if the PR includes a changelog entry (Automatically sets --file ".changes").
+    -u, --url 'TARGET_URL'           Specify the URL to append to add to the GitHub review. Usually you will use an URL to the Jenkins build log.
+    -P                               '--PR 'NUMBER'
+                                     Specify the PR number instead of checking all of them. This will force gitbot to run the against a specific PR number,even if it is not needed (useful for using Jenkins with GitHub webhooks).
```

Comment:
There was a typo here, as well as at lib/opt_parser.rb (last parenthesis was missing). It's now corrected.